impeller/entity/contents/text_contents.cc

```diff
@@ -36,7 +38,7 @@ Color TextContents::GetColor() const {
 }
 
 bool TextContents::CanInheritOpacity(const Entity& entity) const {
-  return !frame_->MaybeHasOverlapping();
+  return false;
```

Comment:
Not sure I follow why this isn't true anymore. Add a comment maybe?

Reply:
comment added